src/App.js

```diff
@@ -1,21 +1,120 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Accordionater, AccordionaterOutput } from './components';
+import AppCss from './App.css'; // eslint-disable-line
 
 class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    );
-  }
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			accordionatorHtmlOutput: [],
+			code: '',
+			accordionTitleForIds: '',
+			headingType: 'h4',
+		};
+
+		this.handleWizzyChange = this.handleWizzyChange.bind(this);
+		this.handleGenerateCodeClick = this.handleGenerateCodeClick.bind(this);
+		this.handleAccordionTitleChange = this.handleAccordionTitleChange.bind(this);
+		this.handleHeadingTypeChange = this.handleHeadingTypeChange.bind(this);
+	}
+
+	handleAccordionTitleChange(accordionTitle) {
+		this.setState({ accordionTitleForIds: this.cleanStringForId(accordionTitle) });
+	}
+
+	handleGenerateCodeClick() {
+		let output = '';
+
+		this.state.accordionatorHtmlOutput.map((outputObject, index) => {
```

Comment:
this is not blocking but would I think this scenario i have been writing something like


```javascript
const outputList = this.state.accordionatorHtmlOutput.map((outputObject, index) => {
    return `<all the out html goes here>`;
};

const output = outputList.join('');

...
```

or something along those lines...what do you think?

Reply:
and i guess maybe if you are feeling crazy, you would move the map function to a named function outside of this block rather than being anonymous